nss-day-cohort-22/nutshell-wiggly-donkeys
# Data base pe

Review code, fetch and test.  
Functionality to test: You should be able to:
1. Enter your name and get an error (we all have an account in users now).
2. Enter a new test name/email and see it in local storage as well as see the id number of the user in session storage.
